### ISSUE TITLE
Change "How To Install"

### ISSUE THREAD

**Issue by roest01** [sitecake/sitecake#68]
I've created the first index.html with code provided by sitecake.com

```
<div class="sc-content">
<h1>Fine looking heading</h1>
<img src=”feature.jpg">
<p>Some text</p>
</div>
```

The Problem is, if you don't have an html > head + body area the login window at sitecake.php does not appear.

Providing a bigger index.html demo code at sitecake.com should solve this for other starters.

**Comment by sitecake** [comment]
You are right. We tried to keep things simple, but for somebody it's too simple. We will add more help for first starters. Thanks
